site/guide/work-with-model-findings.qmd

```diff
@@ -0,0 +1,109 @@
+---
+title: "Work with model findings"
+date: last-modified
+---
+
+Model findings are detailed observations identified during the model validation process, highlighting any major or minor issues, deficiencies, model limitations, stability and robustness concerns, or needed adjustments.
+
+These findings are critical for understanding the risk exposure and compliance status of models within an organization. To make them easier to track, findings are typically categorized by risk area, business unit, model status, and individual model, enabling targeted resolution and informed decision-making to mitigate identified risks and ensure model reliability and accuracy.
+
+## Prerequisites
+
+- You hold the `Validator` role
+- You must be [logged into to the ValidMind UI]({{< var vm_ui >}}).
+- The model is already [registered in the model inventory](register-models-in-model-inventory.qmd)
+- A model document project is completed or in progress
+
+## View model findings
+
+To view all project findings: 
+
+1. In the left sidebar, click [**Model Findings**](https://app.prod.validmind.ai/model-findings).
+
+   In the page that opens, you can see a list of all model findings, including information for:
+
+   - Title
+   - Severity
+   - Status
+   - Inventory model
+   - Risk area
+   - Assignee
+   - Due date
+
+2. Click on any model finding for more information, including the proposed remediation plan.
+
+## Filter project findings
+
+By default, all model findings for all models are displayed and sorted by creation date, with the latest findings displayed first. To narrow down the findings to only those you want to see, you can apply some filters:
+
+1. In the left sidebar, click [**Model Findings**](https://app.prod.validmind.ai/model-findings) and then click {{< fa filter >}} **Filter**.
+
+2. On the **Select your filters** dialog that opens, select your filtering criteria for:
+
+   - Status
+   - Model inventory
+   - Risk area
+   - Business units
+   - Severity
+   - Assignee
+
+   For example: To filter by open findings for the Customer Churn Model assigned to Jane Doe:
+
+   ![](filter.png){width=60% fig-alt="TBD"}
+
+3. Click **Apply Filters**.
+
+Filters can be removed by clicking on the {{< fa xmark >}} next to them on the main [**Model Findings**](https://app.prod.validmind.ai/model-findings) page where the results are displayed. 
+
+## Add project findings
+
+As part of validation process, you may find issues with the model documentation that must be resolved. To indicate that there is an issue and to track the resolution, you add a new finding:
+
+1. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
+
+2. Select a model by clicking on it or find your model by applying a filter or searching for it.
+
+3. In the left sidebar that appears for your model, click **Documentation**.
+
+4. On the page that opens, click **Add Finding**.
```

Comment:
There's actually another, context-sensitive way to add findings via the 'ValidMind Insights' right sidebar that needs to be covered here: 

<img width="386" alt="image" src="https://github.com/validmind/documentation/assets/15148011/b8465e5d-102b-4183-aaa7-d7a594c7c0f2">

Reply:
Addressed via https://github.com/validmind/documentation/pull/155/commits/b4abb235c8f5167df6766bc9c7f3e56132748906